### PR TITLE
Add terms for JIRA ticket #129

### PR DESCRIPTION
**Pull request by anngvu**
- **Merge https://github.com/nf-osi/nf-metadata-dictionary/pull/145 first**
- Pull changes from main into this PR
- Merge this PR
- Create new minor release
- Watch CI/CD workflow triggered with new published release

### CODE REVIEW

File: NF.csv [hunk 1]
```diff
@@ -184,8 +184,8 @@ Illumina NovaSeq 6000,"A DNA sequencer which is manufactured by the Illumina cor
 PacBio Sequel IIe System,PacBio Sequel IIe System,,,FALSE,,platform,,,,PacBio_Sequel_Iie_System,Class,Assay,Platform,Platform,,,
 PacBio Sequel II System,PacBio Sequel II System,,,FALSE,,platform,,,,PacBio_Sequel_II_System,Class,Assay,Platform,Platform,,,
 NanostringGeoMx,NanostringGeoMx,,,FALSE,,platform,,,,Nanostring_GeoMx,Class,Assay,Platform,Platform,,,
-Chromium X,"(From 10x Genomics) A single-cell platform with capability for hundreds to hundreds of thousands of cells in a single run, used for analysis of cell activity in multiple dimensions, including gene expression,
-chromatin accessibility, cell surface proteins, immune clonotype, antigen specificity, and CRISPR perturbation screens.",,,FALSE,https://www.10xgenomics.com/chromium-x,platform,,,,Chromium_X,Class,Assay,Platform,Platform,,,
```
Comment (allaway) [inline NF.csv:188]: Good catch! Not sure when this linebreak was introduced!

File: NF.csv [hunk 2]
```diff
@@ -184,8 +184,8 @@ Illumina NovaSeq 6000,"A DNA sequencer which is manufactured by the Illumina cor
...
 platform,"A sequencing platform, microscope, spectroscope/plate reader, or other platform for collecting data. ","HiSeq3000, HiSeq2500, HiSeq4000, NextSeq500, HiSeq2000, MiSeq, Zeiss LSM 980, Affy5.0, Affy6.0, PacBioRSII, GAIIx, Illumina_HumanOmni1-Quadv1.0, Illumina_1M, Illumina_h650, Illumina_Omni2pt5M, Illumina_Omni5M, Illumina MouseWG-6 v2.0 expression beadchip, Perlegen300Karray, Agilent44Karray, IlluminaWholeGenomeDASL, IlluminaHumanHap300, NanostringnCounter, LTQOrbitrapXL, IlluminaHumanMethylation450, Illumina Infinium MethylationEPIC BeadChip, AffymetrixU133AB, Affymetrix Human Gene 1.0 ST Array, AffymetrixU133Plus2, HiSeqX, Bionano Irys, Infinium HumanOmniExpressExome, NextSeq 550, NextSeq 1000, NextSeq 2000, PacBio Sequel IIe System, PacBio Sequel II System, NanostringGeoMx, LI-COR Odyssey CLx, Spectramax M Series, Olympus DP80, Orbitrap Fusion Lumos Tribrid, Illumina NovaSeq 6000, Chromium X",,FALSE,https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group&anchor=platform,experimentalData,,,,platform,Property,Assay,annotationProperty,,Platform,,
```
Comment (allaway) [inline NF.csv:191]:
```suggestion
platform,"A sequencing platform, microscope, spectroscope/plate reader, or other platform for collecting data. ","HiSeq3000, HiSeq2500, HiSeq4000, NextSeq500, HiSeq2000, MiSeq, Zeiss LSM 980, Affy5.0, Affy6.0, PacBioRSII, GAIIx, Illumina_HumanOmni1-Quadv1.0, Illumina_1M, Illumina_h650, Illumina_Omni2pt5M, Illumina_Omni5M, Illumina MouseWG-6 v2.0 expression beadchip, Perlegen300Karray, Agilent44Karray, IlluminaWholeGenomeDASL, IlluminaHumanHap300, NanostringnCounter, LTQOrbitrapXL, IlluminaHumanMethylation450, Illumina Infinium MethylationEPIC BeadChip, AffymetrixU133AB, Affymetrix Human Gene 1.0 ST Array, AffymetrixU133Plus2, HiSeqX, Bionano Irys, Infinium HumanOmniExpressExome, NextSeq 550, NextSeq 1000, NextSeq 2000, PacBio Sequel IIe System, PacBio Sequel II System, NanostringGeoMx, LI-COR Odyssey CLx, Spectramax M Series, Olympus DP80, Orbitrap Fusion Lumos Tribrid, Illumina NovaSeq 6000, Chromium X,EnVision 2103 Multiplate Reader",,FALSE,https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=read_group&anchor=platform,experimentalData,,,,platform,Property,Assay,annotationProperty,,Platform,,
```

File: NF.csv [hunk 3]
```diff
@@ -377,6 +377,10 @@ cNF98.4c,A primary cutaneous neurofibroma cell line; not broadly available. See
 Prss56Cre;R26mT,"Prss56Cre;R26mT is a transgenic non-human animal model for Neurofibromatosis type 1, wherein the Nf1 gene is specifically inactivated in BC cells and derivatives thereof.","","",FALSE,https://patents.google.com/patent/EP3420811A1/en,modelSystemName,,"","",Prss56Cre;R26mT,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG-Arg816,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG-Arg816,Class,Biosample,Model_System,Mouse_Model,"","",""
+Nf2-/- Schwann cell line,"A Nf2-/- mouse schwann cell line","","",FALSE,"",modelSystemName,,"","",Nf2-/-Schwann,Class,Biosample,Model_System,Cell_Line_Model,"","",""
```
Comment (allaway) [inline NF.csv:380]: I have a couple of requests for these: 
- can you ask Roberta if there are more specific names for these cell lines? These are really vague
- can you ask Roberta if these cell lines are published in the literature or listed on cellosaurus? If so, we should default to the name listed on Cellosaurus. 

e.g. is this one https://web.expasy.org/cellosaurus/CVCL_7691 or https://web.expasy.org/cellosaurus/CVCL_JK03?

File: NF.csv [hunk 4]
```diff
@@ -377,6 +377,10 @@ cNF98.4c,A primary cutaneous neurofibroma cell line; not broadly available. See
...
+hTERT arachnoid,"A human arachnoid cell line that has been used in drug screening (suitable control for hTERT NF2 arachnoid)","","",FALSE,"",modelSystemName,,"","",hTERT_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT NF2 arachnoid,"A NF2 knock-out human arachnoid cell line developed with CRISPR that has been used in drug screening","","",FALSE,"",modelSystemName,,"","",hTERT_NF2_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT meningioma,"A human meningioma cell line that has been used in drug screening","","",FALSE,"",modelSystemName,,"","",hTERT_meningioma_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
```
Comment (allaway) [inline NF.csv:383]: probably https://web.expasy.org/cellosaurus/CVCL_1959 or https://web.expasy.org/cellosaurus/CVCL_JK01 (this lab has used this cell line in the past)

but also possibly
https://web.expasy.org/cellosaurus/CVCL_M426
https://web.expasy.org/cellosaurus/CVCL_M427
https://web.expasy.org/cellosaurus/CVCL_M428

File: NF.csv [hunk 5]
```diff
@@ -377,6 +377,10 @@ cNF98.4c,A primary cutaneous neurofibroma cell line; not broadly available. See
...
 Nf1-OPG,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG-Arg816,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG-Arg816,Class,Biosample,Model_System,Mouse_Model,"","",""
+Nf2-/- Schwann cell line,"A Nf2-/- mouse schwann cell line","","",FALSE,"",modelSystemName,,"","",Nf2-/-Schwann,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT arachnoid,"A human arachnoid cell line that has been used in drug screening (suitable control for hTERT NF2 arachnoid)","","",FALSE,"",modelSystemName,,"","",hTERT_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT NF2 arachnoid,"A NF2 knock-out human arachnoid cell line developed with CRISPR that has been used in drug screening","","",FALSE,"",modelSystemName,,"","",hTERT_NF2_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
```
Comment (allaway) [inline NF.csv:382]: possibly derivative of AC007-hTERT? https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4627286/

File: NF.csv [hunk 6]
```diff
@@ -377,6 +377,10 @@ cNF98.4c,A primary cutaneous neurofibroma cell line; not broadly available. See
 Prss56Cre;R26mT,"Prss56Cre;R26mT is a transgenic non-human animal model for Neurofibromatosis type 1, wherein the Nf1 gene is specifically inactivated in BC cells and derivatives thereof.","","",FALSE,https://patents.google.com/patent/EP3420811A1/en,modelSystemName,,"","",Prss56Cre;R26mT,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG,Class,Biosample,Model_System,Mouse_Model,"","",""
 Nf1-OPG-Arg816,"","","",FALSE,"",modelSystemName,,"","",Nf1-OPG-Arg816,Class,Biosample,Model_System,Mouse_Model,"","",""
+Nf2-/- Schwann cell line,"A Nf2-/- mouse schwann cell line","","",FALSE,"",modelSystemName,,"","",Nf2-/-Schwann,Class,Biosample,Model_System,Cell_Line_Model,"","",""
+hTERT arachnoid,"A human arachnoid cell line that has been used in drug screening (suitable control for hTERT NF2 arachnoid)","","",FALSE,"",modelSystemName,,"","",hTERT_arachnoid,Class,Biosample,Model_System,Cell_Line_Model,"","",""
```
Comment (allaway) [inline NF.csv:381]: possibly https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4627286/ AC007-hTERT?

**Review by allaway — CHANGES_REQUESTED** [review]

File: NF.csv (hunk 2 above)
Comment (allaway) [reply NF.csv]: Oops, sorry for not adding this suggestion to the correct file 🙃

**Comment by anngvu** [comment]
@allaway Not sure if there can be better nomenclature for non-registered lines. How do you feel about adding the PMID to the label in order to better distinguish somewhat generic names?

**Comment by allaway** [comment]
Looks great to me, thanks! I added the new modelsystemnames to the annotationProperty file.

**Comment by allaway** [comment]
Ok, not sure what I did wrong to break the build.... 🤔

**Comment by anngvu** [comment]
Interesting...I removed the portion (PMID:XXXX) based on a hunch and it was fine locally.

**Review by allaway — APPROVED** [review]

**Comment by allaway** [comment]
LGTM, thanks!

**Comment by allaway** [comment]
## NADIA Curation Report

**Synapse Project:** [syn74316154](https://www.synapse.org/#!Synapse:syn74316154)

### Annotations Applied
| Field | Value |
|-------|-------|
| studyName | RNA Sequencing of murine MPNST Tumors treated with myelomodulatory agents |
| diseaseFocus | Neurofibromatosis type 1 |
| manifestation | Malignant Peripheral Nerve Sheath Tumor |
| dataType | geneExpression |
| assay | spatialTranscriptomics |
| species | Mus musculus |
| tumorType | Malignant Peripheral Nerve Sheath Tumor |
| platform | 10x Visium |
| studyLeads | Siddhi N Paudel, Timothy P Cripe |
| institutions | Nationwide Childrens Hospital Columbus |
| fundingAgency | NCI NIH HHS |
| pmid | 39044819 |
| doi | 10.3389/fimmu.2024.1384623 |
| alternateDataRepository | geo:GSE244829, insdc.sra:PRJNA1024972 |

### Curation Notes
- GPL24247 = 10x Genomics Visium spatial transcriptomics platform
- assay corrected to spatialTranscriptomics (not rnaSeq)
- 23 FASTQ files in Raw Data folder
- manifestation corrected: removed Juvenile Myelomonocytic Leukemia (wrong for murine MPNST study)
- studyLeads corrected from LastName-First to FirstName-Last

**Status:** pendingReview ✓

**Pull request merged**